### PR TITLE
Updating readme for cost estimate and errors thrown

### PR DESCRIPTION
**Pull request by ankit819**
|JIRA    | [ENG-1811](https://eligible.atlassian.net/browse/ENG-1811)
|:-------|:-----------------------------------------------------------

### Description

Updating Readme for cost estimate and all the errors thrown from eligible ruby gem.

### Checklists

**Pre-release and code review:**
- [x] CI build is green
- [x] CodeClimate does not see any new issues
- [x] "Ready for review" label attached to PR and reviewers mentioned in a comment
- [x] Changes have been reviewed by at least one Senior Engineer
- [x] JIRA issue has a link to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -704,6 +727,19 @@ response = Eligible::Referral.create(params)
 response.to_hash
 ```
 
+## Errors
+
+This is the list of errors thrown from the eligible ruby gem.
+
+1. EligibleError
+2. APIError
```
Comment (sreehari) [inline README.md]: include module name pls...
Eligible::EligibleError

Comment (sreehari) [inline README.md]: also pls add very brief description about each exception

Comment (ankit819) [inline README.md]: Ok cool, will do that.

**Comment by sreehari** [comment]
:+1:

**Pull request merged**